src/theory/datatypes/theory_datatypes_utils.h

```diff
@@ -146,17 +146,31 @@ bool checkClash(Node n1, Node n2, std::vector<Node>& rew);
  * function mkSygusTerm.
  */
 Kind getOperatorKindForSygusBuiltin(Node op);
+/**
+ * Returns the total version of Kind k if it is a partial operator, or
+ * otherwise k itself.
+ */
+Kind getEliminateKind(Kind k);
+/**
+ * Returns a version of n where all partial functions such as bvudiv
+ * have been replaced by their total versions like bvudiv_total.
```

Comment:
is this example up to date? isn't `bvudiv` already total by now?

Reply:
bvudiv still exists (to support legacy version of smt-lib), although this will be eliminated in the future.